Resources/Prototypes/_EinsteinEngines/Entities/Mobs/Player/silicon_base.yml

```
@@ -330,4 +332,5 @@
     - type: FootPrints
     - type: Carriable
     - type: Targeting
-    - type: LayingDown
\ No newline at end of file
+    - type: LayingDown
+    - type: ExaminableCharacter # WWDP
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)